app/stylesheets/boilerplate/static-content.scss

```diff
@@ -0,0 +1,29 @@
+@import '../layout';
+@import '../config/colours';
+@import '../config/dimensions';
+@import '../config/type';
+
+// homepage
+.homebanner {
+  float: left;
+  height: $full;
+}
+
+.homebanner img {
+  height: $full;
+}
+
+.homecontent  {
+  overflow: hidden;
```

Comment:
there are a few issues with vertical overflow on a small screen. It would be nice if this div had `overflow-y: auto;`, and was the right height so it didn't go behind the footer. tricky.